client/src/components/AddGame.jsx

```diff
@@ -213,12 +216,25 @@ class _AddGame extends React.Component {
   }
 
   render() {
-    let alert = '';
-    if (this.state.gameName) {
-      alert =
-        <div className="alert alert-primary">
-          {this.state.gameName} was added!
+
+    const { user } = this.props;
+
+    // Show loading spinner if fetching user
+    if (user.isFetching && !user.isReceived) {
+      return (
+        <div className="d-flex justify-content-center mt-5">
+          <Spinner animation="border" />
         </div>
+      );
+    }
+
+    // Show login form if not logged in
+    if (!user.isLoggedIn) {
+      return <Login />
```

Comment:
This should be `<Redirect to="/login">`  . Handling it like this will end up rendering Login as a child to AddGame

Reply:
Ok, these have been updated